docs/tre-admins/setup-instructions/pre-deployment-steps.md

```diff
@@ -20,7 +20,7 @@
   !!! tip
       To retrieve your Azure subscription ID, use the `az` command line interface available in the development container. In the terminal window in Visual Studio Code, type `az login` followed by `az account show` to see your default subscription. Please refer to `az account -help` for further details on how to change your active subscription.
 
-2. Comment out the following variables by starting the line with a hash `#`.
```

Comment:
Should we not comment out the variables by default in the sample files and remove this step? (or make it an option to change the az credentials?

Reply:
We seem to promote the manual deployment options so I think: yes, make them commented out by default and remove the step and highlight the fact that they need to be there at https://microsoft.github.io/AzureTRE/tre-admins/setup-instructions/workflows/

Reply:
@tompaana can you verify that this is correct

https://github.com/microsoft/AzureTRE/blob/52086230b4e1ca90edf01d33a0d7262f9a30a7eb/docs/tre-admins/setup-instructions/workflows.md#L53-L65

Reply:
@Lybecker Time has passed since I last tested workflows from-scratch myself. I'd need to re-test to make sure.